docs/stratis.txt

```diff
@@ -75,6 +75,17 @@ daemon redundancy::
 daemon version::
        Show the Stratis service's version.
 
+ENVIRONMENT VARIABLES
+---------------------
+STRATIS_DBUS_TIMEOUT::
+	 Sets a timeout for any Stratis D-Bus call. If this environment variable is
+	 not set, a default value of 120 seconds is used for the timeout.
+	 Accepted STRATIS_DBUS_TIMEOUT environment variable values are:
+         1. an integer between 0 (inclusive) and INT31_MAX (inclusive),
```

Comment:
Someone might object that INT31_MAX is not a widely understood name, we should probably come up with some different way of saying the same thing.